.github/workflows/maven.yml

```diff
@@ -15,7 +15,7 @@ jobs:
           persist-credentials: false
 
       - name: Checkout core
-        run: git clone --depth=50 --branch=develop https://github.com/informatici/openhospital-core.git openhospital-core
+        run: git clone --depth=50 --branch=OP-1165-fix-circular-reference-in-jpa-config https://github.com/informatici/openhospital-core.git openhospital-core
```

Comment:
doesn't this need to be changed before merge?

Reply:
yes, of course, if the change to `src/main/java/org/isf/config/SecurityConfig.java` is also neeed, otherwise we can just close this PR becuase the problem was actually in core only.